src/Mailcode/Commands/Command/ShowSnippet.php

```diff
@@ -18,29 +18,35 @@
  * @subpackage Commands
  * @author Sebastian Mordziol <s.mordziol@mistralys.eu>
  */
-class Mailcode_Commands_Command_ShowSnippet extends Mailcode_Commands_ShowBase implements Mailcode_Interfaces_Commands_Validation_NoHTML
+class Mailcode_Commands_Command_ShowSnippet
+    extends Mailcode_Commands_ShowBase
+    implements
+    Mailcode_Interfaces_Commands_Validation_NoHTML,
+    NamespaceInterface
 {
     use Mailcode_Traits_Commands_Validation_NoHTML;
+    use NamespaceTrait;
 
-    public function getName() : string
+    public function getName(): string
     {
         return 'showsnippet';
     }
-    
-    public function getLabel() : string
+
+    public function getLabel(): string
     {
         return t('Show text snippet');
     }
-    
-    protected function getValidations() : array
+
+    protected function getValidations(): array
     {
         return array(
             Mailcode_Interfaces_Commands_Validation_Variable::VALIDATION_NAME_VARIABLE,
-            Mailcode_Interfaces_Commands_Validation_NoHTML::VALIDATION_NAME_NOHTML
+            Mailcode_Interfaces_Commands_Validation_NoHTML::VALIDATION_NAME_NOHTML,
+            'check_namespace'
```

Comment:
Since there is already a `NamespaceInterface`, it would be good to have this name there in a constant, like `CountInterface::VALIDATION_COUNT_NAME`.